core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -177,6 +182,21 @@ var ContributorDashboardAdminPage = function() {
       'Submit Question Right Element taking too long to appear');
     expect(await contributionRights.getText()).toBe('Allowed');
   };
+
+  this.checkstatslistelements = async function(number) {
+    await waitFor.presenceOf(statsTable, 'stats table was not visible');
+    var statsListItems = await statsListItemsSelector();
+    expect(statsListItems.length).toBe(number);
+  };
+
+  this.checkexpandablestats = async function() {
```

Comment:
what are we checking here?

Reply:
We are checking if the row is expanded on click or not.

Reply:
so maybe reanme it to epectStatsRowsAreEpanded().

Note: The `expect` prefix is used for testing blocks.

Reply:
Sg. Updated.